README.md

```diff
@@ -36,11 +36,17 @@ TODO: list all components used to build current Ship version. Make a release aft
 
 Just fork or clone and push repository into your own repo. Rename [SHIP_README.md](SHIP_README.md) into README.md and optionally remove all files in the repo root except: `.drone.yml`, `docker-compose.drone-tests.yml`, `dodcker-compose.yml`.
 
+## Monitoring
```

Comment:
I think all grafana monitoring description need to reside in the ansible-grafana repository. We need to do a couple of things: 
1. Move this description and all notes about grafana configuration to the ansible-grafana repository. That would be nice to include screenshot there, so users can see what they get. In the description, please include notes about what we monitor at the moment (e.x.: cpu, memory, docker containers, etc)
2. Let's make release of the repository using github releases, see (node-mongo)[https://github.com/paralect/node-mongo/releases] for example.

In Ship we only need to keep a reference to ansible-grafana in Ship Demo section and also in Ship Components. For the first version let's do a able with two columns: name (must be a link to the repository), version (must be a link to the release used in ship) and description — a one-sentence description of the component.